dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -510,7 +510,13 @@ def _test_logged_in_milestone_with_image(async_activity_writes:)
     expect_s3_upload
 
     original_activity_count = Activity.count
-    assert_creates(LevelSource, UserLevel, LevelSourceImage) do
+    original_user_level_count = UserLevel.count
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `original_user_level_count`.